src/autopas/tuning/AutoTuner.cpp

```diff
@@ -280,6 +287,16 @@ void AutoTuner::addMeasurement(long sample, bool neighborListRebuilt) {
 void AutoTuner::bumpIterationCounters() {
   ++_iteration;
   ++_iterationsSinceTuning;
+  if (inTuningPhase()) {
+    ++_lastTuningIterations;
+    if (_lastTuningIterations == _tuningInterval + 1) {
+      // The tuning interval triggered a new tuning phase during a running tuning phase -> continue tuning ("merge" of
+      // two tuning phases)
```

Comment:
```suggestion
      // The tuning interval triggered a new tuning phase during a running tuning phase.
      // -> continue tuning ("merge" of two tuning phases)
```

Reply:
done in [b9e4111](https://github.com/AutoPas/AutoPas/pull/943/commits/b9e4111c93ce42af332b045755214316b102b040)